### PR TITLE
✨ New Rule: Do you use Prefixes to improve code review communication?

### PR DESCRIPTION
**Pull request by tomek-i**
PBI: https://dev.azure.com/ssw/SSW.Induction/_workitems/edit/85320

## Reason for change (Issue, Email, conversation + reason, etc)
- Relates to rule: https://www.ssw.com.au/rules/prefixes/
- It helps bring structure into Code Review comments which can be machine parse-able
- It assists with clear communication and intend during a code review process

By using prefixes we can prime the reader on the subject matter and also make our comments machine parse-able.

We can easily use prefixes to differentiate between a nitpick eg. personal preferences or something that we deem is an issue and needs to be actioned asap.

Over the shoulder review from: @GordonBeeming, @ncn-ssw, @matt-goldman, @wicksipedia

### CODE REVIEW

**Comment by tomek-i** [comment]
Hm its complaining about the Author but this was pre-filled by netlify ... is it supposed to be my github handle?

File: rules/use-prefixes-to-improve-code-review-communication/rule.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+**issue**: We must address this security vulnerability before merging.
+
+There is a potential for SQL Injection and this vulnerability could lead to a critical security breach if not fixed.
+```
+
+::: good
+Figure: The prefix "issue" and following context clearly define the comment's importance and category.
+:::
+
+In this example, the prefix "issue" convey the urgency of the matter, helping the team prioritize and act accordingly.
```
Comment (ncn-ssw) [inline rules/use-prefixes-to-improve-code-review-communication/rule.md:55]: **issue**: Grammar.
```suggestion
In this example, the prefix "issue" conveys the urgency of the matter, helping the team prioritize and act accordingly.
```

File: rules/use-prefixes-to-improve-code-review-communication/rule.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+@bob-northwind
+This code could be better optimized
+```
+
+::: bad  
+Figure: Without a prefix, this comment's intent is vague. It's not evident whether it's a suggestion, a question, or an issue.
+:::
+
+```plain
+@developer
```
[the remark this answers is not in the document]
Comment (tomek-i) [reply rules/use-prefixes-to-improve-code-review-communication/rule.md:31]: yes, I did not pick that up =/ my bad, will fix it now

File: rules/use-prefixes-to-improve-code-review-communication/rule.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+---
+type: rule
+title: Do you use prefixes to improve code review communication?
+uri: use-prefixes-to-improve-code-review-communication
+authors:
+  - title: Tom Iwainski
```
Comment (wicksipedia) [inline rules/use-prefixes-to-improve-code-review-communication/rule.md:6]:
```suggestion
  - title: Tom Iwainski
    url: https://www.ssw.com.au/people/thomas-iwainski/
```

**Review by matt-goldman — APPROVED** [review]
LGTM with the fix Wicksy suggested

**Review by ncn-ssw — APPROVED** [review]

**Pull request merged**